src/styles/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-size;
+  width: $calendar-size;
+  padding: 10px;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    box-sizing: border-box;
+    height: $column-size;
+    width: $column-size;
+
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    background-color: #eee;
+    border: 1px solid black;
+
+    &:hover {
+      transition-property: transform, background-color;
+      transition-duration: 0.5s;
+      transform: translateY(-20px);
+      cursor: pointer;
+      background-color: $hover-color;
```

Comment:
The `transition` property should be placed outside of the `:hover` selector to avoid redefining it every time the element is hovered over. This can improve performance.